src/components/Avatar/Avatar.story.js

```diff
@@ -0,0 +1,357 @@
NnHs9toGkkvrgblt+6gAySxrRoNZs5JdhjmT1ZOVKPs9ntdG4RuNRvNo3ykqSOeB5VcTcWW29EubaWESAMhYZyD0NI+s1Q4h2geKWIFGB8jUvbwwDMh/vVVo9yl0QkODnny8aL1SSO0fEoAIG7n4UnrRb+g+PiGzz2QhmdunJcZqs1a0hv4HMJKh/AjmjeGapYuOrSyvlieJh0IJGMjz6U5W9xZ63aC7tyuXHUfkaEtoVfFmCcRQNFcNuXa4OGHkapIc7wngSGX31qHtB0DZEL1UOH7rgefnWXxxus6AjKhhzxT43aJ5EN9jGTjOKmW1klvhs6g5rjTFYlefjV3pMYOpvkZxWoxWW0NhdXcSKcKMfu1OOEI0g3s5znzq4t8gKADVo6ZtflUW6DHZVeynT2updX1QlI4ri7IRAMkY5n86fdTuoNIj7Ujk3UeNYVwNxPd6Hf3dr9pZbaRssnhu86aNW48s543iVt5x41XDD1XoOR+V8Qifi97i8vmj7kQIVQevTrWE8TXTXN7K7nLO7OfiT/im1tUZ7iYKcbyTikXV333cp94Hyrp+bXkMLptm1cHaCknB+nJNGpLRBwGXPXnVnPw0wUsYIAvmRmjeHLlG4b06SPo1vGR/wBoqa61B7iVbaLDP5Z5Cszo2xTOOGdNSC8GxFCou0BRgVZcQaP+kUO1VJIwQ3lU+iRiKQbyA3iR0FH3cjxQSSBNyDnn0ppRVAV2Jltw1D9oRp7GOR0wFdgCRjp1pm0rh+20yJjbxmIMdxQHu591TWlxDMFdSCKsnlBTIPIUriqObfBV42tVl4cmyBkYNZRY3g0+zvITGrQXMbRyqVHM/hb3g1r/ABMTc6eLdfxsAayniSyawjW3IB3ymRSB4eXzpI7dINUrItIXM0YPxpg0ZVGrT56DFLdo7W5VwPWira+kjuWkU4LVuezz0q6a1YJBJGpOM1NfFI7ZtpHKkay1ORYlIl5++p7jVpWhO6T61Nw2G0ZVE5XUpOZ51YCMN3iBVdburX7nNWRfIxVsbqCQrW7AYv8AczN5DApYv/27HOc04sgWNiPGlW+i+8f31Ktll+psPA141xwZppRs7YzEfQqSKKjuopLmeDtY1kRiNu7DH1+dIPsw18W0s2jzNgSN2sGf3vxL8Rz+dOl5psEuqR6gEBkik7QA9DywQfSsslUzZjlcS9sr27UFI8HwDE9Ku7a0uZYmM168oYfqE8vpVZaxaDNtk3S27F8FDkflkYo2SDT1T/StM5yQGdioPP5nlXOND3/C/wAAreV7HUHhyQOuPL1pghuWfCnxGaVNO0Y2GozzPcSzmfmTI2cc+gHgBTIJFiHaEgADxqdtaBLZS8W8QQ6Els7wiZpGOFLhQAB1/Ks3vb6fXb/t2XAJwox0A8APKp+JtTXXtaNxjMEP3cOfLPNviamsY1SMEKMmtWLHStmTJkb0iT7MgQLjoMUNJagdKPYioW51czgXZyqO65FcO9xjBY4o/ltNRkAjpRsNCdFp7xTdoH51YKr+NSgZqVY+WTXUAHkB2Uu3CbxuPqD86Zp1xETS/NHtkZc8iW/xSvpWPCgWSS1vBJC5SSJ8ow6gjxraeEdcj4j03tOS3MeFmTyPmPQ1jV2m26k5fizRnD2uXXDusLd23eU92SInk6+X+ankhasbHLyzeY7NhJ3Syg9QDyq7srPZzPXzNUOlcQWGoW0cqP2bMASsnIirldThVQEbe38NZW1RtbbVB80cceCevhQ86Ga3dfAqa8iEly296KCY7tJdcFZivZncc+BxVvaHEYFXnFHCBsrV9U0xHez3ETx/rNC3n/L6+FL9u33Yr0Vw8+XaCXbFRE14z1xuogO88q8zyrnNeZ5Vxwfa8B69Lzltkt185pAPoM1bjgSGG0Z7i9d5AM4iUBQfj1rUWiDqQfKl3VnWC3DuQqbzuJ6YANehjxQfSEnIyTibS4dD0vtpLovJNMI4UxgkdWJ+lK12oJ3D8QDA/wDPhQ/FevTa5rzXG49hG22BfAID/WpYZRNYBif2RKt7v/lYsvn38eGmF1sqtTi/1II6MgNQQ2by3MQVSQyk8h5VcXkG6NXxkgn4irThBIv0pD2iB9jMm0+Icbf61F8orFK7Gjh22xbpy6CnSzjUbe7XEnDb6LebYwWtJOcL9eX7p9RR0UJAHKsUotOma4tNWiwgbC8qnQEnPlUVvGSPpRNwDDbEhSznkoHiT0oKLYkmFcJt9obVMgNG1wVweYPdHhSzxL7OZ47hrnQo1aFubW5bBU/w58PSnnh7TF0qwWIhe1Y75WHix61cNjFehFeYpGLI05No/OF9Z3VhOYLy3kglH4ZFwT/ehc1+idT02z1K37G8to508nXOPd5Ul3fsx0uV2Nvc3NvuPdXIZR8+f1p/N8EsyvNfE066j7M9Uto3ksriK8CDOzBRz7h0PzpJkR4naORGR1OGVhgg+RoNUE37cAcnwrM/atqLWPDcgjYiSV+zXHgD1+madNX1m1098tJuk69mvMmsQ9oPEcutTwwm3MUAkyNzZLePStjl5i2DzbEKaDY0mB3QQPjRtm+x54yMLJHux9M186Y7ONucg77e81FCQbp2/eQj/wAaxFiwt5AbBGbrEQvv8KK0eddO162lxmJmVh64OaAsucawN1ZDu9//AA1yrEIB+JGyuPDxx+fyoUNF0fpTTbqeVGW4RZo5SG2sOR8seVHz6XF2DzQK6bRko3MAehqs9ntwmscIWk7d54xsz7un/PSm1o8wyA5yUI+lPKEZRt9F9OMqQv28axr61aWdisuy5lXO05jB/Og7O3a8uAg/ZJgyHzHlTGHjhXDuqj1PhUccK2Nkn9I9zgc/KvVcHxHzqCWaNrd2Qhu6cYoeG0QBMHqBV6sgHyAFagI3KQeoqQpsjUZrkc1LeOKKOPk/U3HxpI464Pi1S1m1WyVlv40yyL0mA/8AbHjTtuxEPWuQcjFGrOP/2Q=='; // eslint-disable-line
```

Comment:
We can definitely use something like `image-loader` for webpack to store this file externally.

Reply:
We absolutely could use `image-loader` for images, this is ONLY here because we wanted to explicitly test an data URI image (as a one-off).  With that in mind would you still like this to be moved?